MuteSpotifyAds/SpotifyManager.swift

```diff
@@ -8,6 +8,21 @@
 
 import Cocoa
 
+func shell(launchPath: String, arguments: [String]) -> String
```

Comment:
I would rename it to `runCommand(...)`. But this function should not be needed, see below. You can remove it.

Reply:
Ok. I think it could be useful for new features